.github/workflows/ci.yml

```diff
@@ -3,43 +3,46 @@ name: node_CI
 on:
   push:
     branches:
-      - master
+    - master
   pull_request:
     branches:
-      - "**"
+    - '**'
```

Comment:
While we are here we can just skip these redundant steps and just mention `pull_request:` as it would do the same

Reply:
These aren't blockers and I will approve the PR anyway